crates/router/src/configs/settings.rs

```diff
@@ -159,6 +159,7 @@ where
 #[cfg(feature = "dummy_connector")]
 #[derive(Debug, Deserialize, Clone, Default)]
 pub struct DummyConnector {
+    pub is_enabled: bool,
```

Comment:
We could have named this as `enabled` to keep it consistent with the rest of our application config values (take console or file logging configuration for example).